deploy/manifests/base/kong-ingress-dbless.yaml

```diff
@@ -92,6 +96,8 @@ spec:
           value: "true"
         - name: CONTROLLER_PUBLISH_SERVICE
           value: "kong/kong-proxy"
+        - name: CONTROLLER_CONTROLLER_UDPINGRESS
+          value: "false"
```

Comment:
Does it mean : UDPIngress is not supported by default ?

Reply:
Line 94: The hardcode 127.0.0.1:8444 is resolved in PR
https://github.com/Kong/kubernetes-ingress-controller/pull/1487

Reply:
I don't think we need change this mechanism when we deploying both controller and proxy into the same node. either 127.0.0.1:8444 or localhost:8444 would work fine.